app/config/variables.php

```diff
@@ -846,10 +846,11 @@
             ],
             [
                 'name' => '_APP_FUNCTIONS_MAINTENANCE_INTERVAL',
-                'description' => 'Interval how often executor checks for inactive runimes. The default value is 60 seconds.',
-                'introduction' => '1.2.0',
```

Comment:
The introduction should be `1.2.0` itself

Reply:
I'm pretty sure it was introduced in [1.4.0](https://github.com/appwrite/appwrite/blob/e82c22ac1eca532fd83129d39a134a6b2ccf898e/app/config/variables.php#L847). You won't find it in [1.3.8](https://github.com/appwrite/appwrite/blob/1.3.8/app/config/variables.php).

Reply:
Alright thanks for confirming 👍